src/command_modules/azure-cli-find/tests/test_find.py

```diff
@@ -52,10 +52,10 @@ def test_search_index(self):
         six.assertRegex(
             self,
             execute('find -q keyvault list'),
-            'az keyvault certificate list-versions')
+            'az keyvault list')
 
     def test_search_reindex(self):
         six.assertRegex(
             self,
             execute('find -q keyvault list --reindex'),
-            'az keyvault certificate list-versions')
+            'az keyvault list')
```

Comment:
This was due to https://github.com/Azure/azure-cli/issues/2805.
Now that's fixed, I should be able to remove this :)